atr/post/distribution.py

```diff
@@ -17,7 +17,7 @@
 
 from __future__ import annotations
 
-import quart
+from typing import cast
```

Comment:
Per the code conventions, we do not use `cast` or type ignores unless there is strictly no other alternative.